vtex-apps/modal-layout
# Fix load-session trigger not working

#### What problem is this solving?

Trigger `load-session` not working when another modal renders before the `load-session` modal.

#### How to test it?



[Workspace](https://klynger--coolboxpe.myvtex.com/)
[Workspace with the problem]( https://sebastest--coolboxpe.myvtex.com/)

1. Open the workspace
2. you should see the modal
3. If you want to test again just open the dev tools and paste this line of code `sessionStorage.removeItem('hasOpenedModal')`

#### Related to / Depends on

Fixes https://github.com/vtex-apps/store-discussion/issues/427

#### How does this PR make you feel? [:link:](http://giphy.com/)



![](https://media.giphy.com/media/THCXwZUTU0M3mXoFCO/giphy.gif)
